slides/02_App_Setup.html

```diff
@@ -83,15 +83,23 @@ <h2>JSX</h2>
             <h2>JSX</h2>
             <p>You can use any JS expression within curly braces.</p>
             <pre>
-                    <code class="hljs javascript" data-trim>
+                    <code class="hljs javascript" data-trim><script type="text/template">
                         const name = 'John Wayne';
 
                         const header = <h1>Hello, {name}</h1>;
 
                         const math = <p>2 + 2 equals {2 + 2}</p>;
 
                         const formatted = <h1>Hello, {formatName(name)}</h1>;
-                    </code>
+
+                        const image = (
+                          <img
+                            src="./images/profile.png"
+                            width={800}
+                            height={800 * 0.75}
+                          />
+                        );
```

Comment:
Code samples with img tags should follow accessability best practice and include the alt tag
```suggestion
                        const image = (<img
                            src="./images/profile.png"
                                                        alt="Profile picture"
                            width={800}
                            height={800 * 0.75}
                          />);
```
(reduced line breaks in sample as the page isn't high enough otherwise)

Reply:
Absolutely right. I'll add that.